packages/patternfly-4/react-core/src/components/Select/Select.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import SingleSelect from './SingleSelect';
+import SelectToggle from './SelectToggle';
+import { SelectContext } from './selectConstants';
+
+// seed for the aria-labelledby ID
+let currentId = 0;
+
+const propTypes = {
+  /** Content rendered inside the Select */
+  children: PropTypes.node,
+  /** Classes applied to the root of the Select */
+  className: PropTypes.string,
+  /** Flag to indicate if select is expanded */
+  isExpanded: PropTypes.bool,
+  /** Placeholder text of Select */
+  placeholderText: PropTypes.string,
+  /** Array of SelectOption nodes that will be rendered */
+  selectOptions: PropTypes.array,
+  /** Selected item(s) structure */
+  selections: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```

Comment:
@srambach For the Single selection variation will the selection always be the text string?

Reply:
Is it an array of strings? Could be more specific, e.g. `PropTypes.arrayOf(PropTypes.string)`